src/mmw/mmw/settings/base.py

```diff
@@ -57,7 +57,9 @@ def get_env_setting(setting):
 
 
 # FILE STORAGE CONFIGURATION
-DEFAULT_FILE_STORAGE = 'django.core.files.storage.FileSystemStorage'
+DEFAULT_FILE_STORAGE = environ.get('MMW_DEFAULT_FILE_STORAGE', 'storages.backends.s3boto3.S3Boto3Storage')
+AWS_STORAGE_BUCKET_NAME = environ.get('MMW_AWS_STORAGE_BUCKET_NAME', 'mmw-development-data-us-east-1')
```

Comment:
I recall discussing the potential for using `MMW_STACK_TYPE` to derive an environment specific S3 bucket name vs. creating another variable that needs to be set via `cloud-config`. Is there anything with the former approach that didn't work out?

Reply:
I recall discussing that too but it was absent when I was drafting this PR. I think that that approach is cleaner so I decided to go with it instead. 

The only drawback is we'd need to cut a release in order to update bucket names, etc, but with the rigidity of CloudFormation I don't think that adds much additional effort.